src/Libraries/Microsoft.Extensions.Hosting.Testing/README.md

```diff
@@ -1 +1,52 @@
-README
+# Microsoft.Extensions.Hosting.Testing
+
+Tools for integration test hosting and related test oriented helpers.
```

Comment:
.. of ASP.NET Core apps?

also is it worth mentioning the Microsoft.AspNetCore.Testing package here, and vice versa in that readme?

Reply:
actually not necessarily of ASP.NET Core apps,  but of apps built with Microsoft.Extensions.Hosting. Anyway -- it should say it's not totally generic.